docs/superpowers/plans/2026-05-07-sdk-0.1.76-upgrade.md

```
@@ -0,0 +1,627 @@
+
+可用以下命令验证当前内容：
+
+```bash
+grep -n "claude-agent-sdk" pyproject.toml
+```
+
+预期：输出 `"claude-agent-sdk>=0.1.73",`（升级前）。
+
+用 Edit 工具改成：
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)